PWGHF/D2H/Tasks/taskDs.cxx

```diff
@@ -58,7 +58,13 @@ struct HfTaskDs {
   Configurable<std::vector<int>> classMl{"classMl", {0, 2, 3}, "Indexes of ML scores to be stored. Three indexes max."};
   Configurable<double> yCandGenMax{"yCandGenMax", 0.5, "max. gen particle rapidity"};
   Configurable<double> yCandRecoMax{"yCandRecoMax", 0.8, "max. cand. rapidity"};
+  Configurable<double> minMassDsSignal{"minMassDsSignal", 1.934, "min mass for Ds signal"};
+  Configurable<double> maxMassDsSignal{"maxMassDsSignal", 1.994, "max mass for Ds signal"};
+  Configurable<double> minMassDplusSignal{"minMassDplusSignal", 1.866, "min mass for Dplus signal"};
+  Configurable<double> maxMassDplusSignal{"maxMassDplusSignal", 1.906, "max mass for Dplus signal"};
```

Comment:
`min`, `max` are attributes of the quantity `Mass`, so put them as suffixes, not prefixes.